client/task-list/index.js

```diff
@@ -187,56 +198,20 @@ class TaskDashboard extends Component {
 		return currentTask;
 	}
 
-	renderPrompt() {
-		if ( this.props.promptShown ) {
-			return null;
-		}
-
-		return (
-			<Snackbar className="woocommerce-task-card__prompt">
-				<div className="woocommerce-task-card__prompt-pointer" />
-				<div className="woocommerce-task-card__prompt-content">
-					<span>
-						{ __( 'Is this card useful?', 'woocommerce-admin' ) }
-					</span>
-					<div className="woocommerce-task-card__prompt-actions">
-						<Button
-							isLink
-							onClick={ () => this.hideTaskCard( 'hide_card' ) }
-						>
-							{ __( 'No, hide it', 'woocommerce-admin' ) }
-						</Button>
-
-						<Button isLink onClick={ () => this.keepTaskCard() }>
-							{ __( 'Yes, keep it', 'woocommerce-admin' ) }
-						</Button>
-					</div>
-				</div>
-			</Snackbar>
-		);
-	}
-
 	renderMenu() {
 		return (
-			<EllipsisMenu
-				label={ __( 'Task List Options', 'woocommerce-admin' ) }
-				renderContent={ () => (
-					<div className="woocommerce-task-card__section-controls">
-						<MenuItem
-							isClickable
-							onInvoke={ () =>
-								this.hideTaskCard( 'remove_card' )
-							}
-						>
-							<Icon
-								icon={ 'trash' }
-								label={ __( 'Remove block' ) }
-							/>
-							{ __( 'Remove this card', 'woocommerce-admin' ) }
-						</MenuItem>
-					</div>
-				) }
-			/>
+			<div className="woocommerce-card__menu woocommerce-card__header-item">
+				<EllipsisMenu
+					label={ __( 'Task List Options', 'woocommerce-admin' ) }
+					renderContent={ () => (
+						<div className="woocommerce-task-card__section-controls">
+							<Button onClick={ () => this.hideTaskCard( 'remove_card' ) }>
```

Comment:
Another unrelated comment: It always bugged me that clicking this didn't cause the task list to disappear immediately or at least show a spinner.

Reply:
OK to handle this in a follow up?